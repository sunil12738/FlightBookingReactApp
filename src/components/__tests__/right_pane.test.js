import {shallow} from 'enzyme';
import React from 'react';
import RightPane from '../right_pane.jsx';

test('Link changes the class when hovered', () => {
	const component = shallow(<RightPane />);
	expect(component).toMatchSnapshot();
});