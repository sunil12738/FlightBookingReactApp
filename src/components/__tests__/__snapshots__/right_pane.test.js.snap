// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link changes the class when hovered 1`] = `
ShallowWrapper {
  "complexSelector": ComplexSelector {
    "buildPredicate": [Function],
    "childrenOfNode": [Function],
    "findWhereUnwrapped": [Function],
  },
  "length": 1,
  "node": <div
    className="border-1px-solid-grey width-650px margin-10px padding-10px"
>
    <Itinerary
        data={Array []}
        oneWay={true}
    />
    <FlightDetails
        data={Array []}
        oneWay={true}
    />
</div>,
  "nodes": Array [
    <div
      className="border-1px-solid-grey width-650px margin-10px padding-10px"
>
      <Itinerary
            data={Array []}
            oneWay={true}
      />
      <FlightDetails
            data={Array []}
            oneWay={true}
      />
</div>,
  ],
  "options": Object {},
  "renderer": ReactShallowRenderer {
    "_instance": ShallowComponentWrapper {
      "_calledComponentWillUnmount": false,
      "_compositeType": 0,
      "_context": Object {},
      "_currentElement": <RightPane />,
      "_debugID": 1,
      "_hostContainerInfo": null,
      "_hostParent": null,
      "_instance":  {
        "_reactInternalInstance": [Circular],
        "context": Object {},
        "getData": [Function],
        "props": Object {},
        "refs": Object {},
        "state": Object {
          "data": Array [],
          "oneWay": true,
        },
        "updater": Object {
          "enqueueCallback": [Function],
          "enqueueCallbackInternal": [Function],
          "enqueueElementInternal": [Function],
          "enqueueForceUpdate": [Function],
          "enqueueReplaceState": [Function],
          "enqueueSetState": [Function],
          "isMounted": [Function],
          "validateCallback": [Function],
        },
      },
      "_mountOrder": 1,
      "_pendingCallbacks": null,
      "_pendingElement": null,
      "_pendingForceUpdate": false,
      "_pendingReplaceState": false,
      "_pendingStateQueue": null,
      "_renderedComponent": NoopInternalComponent {
        "_currentElement": <div
          className="border-1px-solid-grey width-650px margin-10px padding-10px"
>
          <Itinerary
                    data={Array []}
                    oneWay={true}
          />
          <FlightDetails
                    data={Array []}
                    oneWay={true}
          />
</div>,
        "_debugID": 2,
        "_renderedOutput": <div
          className="border-1px-solid-grey width-650px margin-10px padding-10px"
>
          <Itinerary
                    data={Array []}
                    oneWay={true}
          />
          <FlightDetails
                    data={Array []}
                    oneWay={true}
          />
</div>,
      },
      "_renderedNodeType": 0,
      "_rootNodeID": 0,
      "_topLevelWrapper": null,
      "_updateBatchNumber": null,
      "_warnedAboutRefsInRender": false,
    },
    "getRenderOutput": [Function],
    "render": [Function],
  },
  "root": [Circular],
  "unrendered": <RightPane />,
}
`;
